Assets/Scripts/UI/PauseMenu.cs

```diff
@@ -0,0 +1,53 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class PauseMenu : MonoBehaviour
+{
+    public static bool isGamePaused = false;
+
+    [SerializeField] public GameObject pauseMenu; 
+    
+    void Start () 
+    {
+        Debug.Log("Hello world!");
+    }
+    private void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Escape))
+        {
+            if (isGamePaused)
+            {
+                ResumeGame();
+            }
+            else
+            {
+                PauseGame();
+            }
+        }
+    }
+
+    public void ResumeGame()
+    {
+        pauseMenu.SetActive(false);
+        Time.timeScale = 1f;
+        isGamePaused = false;
+    }
+    
+    public void PauseGame()
+    {
+        pauseMenu.SetActive(true);
+        Time.timeScale = 0f;
+        isGamePaused = true;
+    }
+
+    public void LoadMenu()
+    {
+        SceneManager.LoadScene("Menu");
+        Time.timeScale = 1f;
```

Comment:
why do you set the timescale here to 1? if we are in the menu..

Reply:
The timeScale is set to 0 when the pause menu is clicked. When you go back to the Menu the timescale should be set to normal